cmd/otk-gen-partition-table/main.go

```diff
@@ -183,21 +184,25 @@ func makePartitionTableFromOtkInput(input *Input) (*disk.PartitionTable, error)
 		if err != nil {
 			return nil, err
 		}
-		pt.Partitions = append(pt.Partitions, disk.Partition{
-			Size: uintSize,
-			UUID: part.PartUUID,
-			Type: part.PartType,
-			// XXX: support lvm,luks here
-			Payload: &disk.Filesystem{
+		// XXX: support lvm,luks here
+		newPart := disk.Partition{
+			Size:     uintSize,
+			UUID:     part.PartUUID,
+			Type:     part.PartType,
```

Comment:
Should we add some (inline) documentation what the difference between "Type" and "PartType" is?
This is actually the same, just to distinguish the two options, right?

Reply:
At the very least we'll have to decide when we bring the current documentation in otk up to par again with the state of the repository; I was planning on doing that when we start distributing otk.